docs/pages/docs/0-get-started/grakn-server.md

```diff
@@ -40,7 +40,7 @@ We recommend changing this to another location to make upgrading Grakn easier an
 
 ### Grakn Cluster
 
-Grakn's clustering functionality is part of [Grakn KBMS](https://grakn.ai/kbms).
+Grakn's clustering functionality is part of [Grakn KGMS](https://grakn.ai/kbms).
 
```

Comment:
maybe https://grakn.ai/grakn-kgms ?

Reply:
fixed in https://github.com/graknlabs/grakn/pull/2681/commits/2c70b33f67c420630da4129ab64f1c26bcf7c58c